### ISSUE TITLE
Angular 6 and rxjs 6 support

### ISSUE THREAD

**Issue by muuvmuuv** [DethAriel/ng-recaptcha#107]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [x] feature request
- [ ] question
```

### OS and Version?
MacOS High Sierra 10.13.4

### Versions
```
Angular CLI: 6.0.0
Node: 10.1.0
OS: darwin x64
Angular: 6.0.0
... animations, cli, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.0
@angular-devkit/build-angular     0.6.0
@angular-devkit/build-optimizer   0.6.0
@angular-devkit/core              0.6.0
@angular-devkit/schematics        0.6.0
@angular/cdk                      6.0.1
@angular/material                 6.0.1
@ngtools/webpack                  6.0.0
@schematics/angular               0.6.0
@schematics/update                0.6.0
rxjs                              6.1.0
typescript                        2.7.2
webpack                           4.6.0
```

### Repro steps
Tried to update from ng5.2 to 6

### The log given by the failure
```
ERROR in ./node_modules/ng-recaptcha/recaptcha/recaptcha-loader.service.js
Module not found: Error: Can't resolve 'rxjs/BehaviorSubject' in '/Users/marvinheilemann/Documents/Projects/M8FINDER/www/application/client/node_modules/ng-recaptcha/recaptcha'
```

**Comment by Naveentp** [comment]
Any update?

**Comment by Naveentp** [comment]
Temporarily you can use `"rxjs-compat": "^6.1.0"` to have backward compatibility

**Comment by DethAriel** [comment]
@muuvmuuv and all:

Angular v6 support is now available as a beta in `ng-recaptcha@4.0.0-beta.1`. I want to include one or two small features into v3 branch yet, but in the meanwhile you can use the beta version and upgrade happily to Angular 6.

Please submit a new issue if you encounter any problems with that.

Thank you!
